config/configuration.yml.example

```diff
@@ -94,6 +94,12 @@
 #
 # === More configuration options
 #
+# Disable default module:
+#
+# By default user may choose which modules can be disabled
+# this can be done in lib/open_project/configuration.rb
+# disabled_modules:[list_of_modules_name]
```

Comment:
Currently you would not be able to override the module list via environment variable. I mean yes, you would. But only with a single value. So either we use a comma separated list as its value or we enhance `OpenProject::Configuration` so that it can deal with arrays.

Something along the lines of:

```bash
FOO=$'a\nb' ruby -e 'puts ENV.fetch("FOO").split("\n").inspect'
# => ["a", "b"]

FOO=`ls` ruby -e 'puts ENV.fetch("FOO").split("\n").inspect'
# => ["README.md", "bin", "build"]
```

Don't know if/how that would work with the elastic beanstalk configuration, though.